cs/src/core/Bond.cs

```diff
@@ -3,7 +3,39 @@
 
 namespace Bond
 {
-    using System.Collections.Generic;
+    using System;
+    using System.Collections.Generic;
+
+    public partial class GUID

```

Comment:
we have carriage return in this file too, you may have misconfigured your git check this out https://help.github.com/articles/dealing-with-line-endings/
or you could replace the extra line endings with powershelgl
https://github.com/PowerShell/Win32-OpenSSH/wiki/Dos2Unix---Text-file-format-converters